src/Raven.Client/Documents/Operations/OperationExecutor.Operations.cs

```diff
@@ -1,13 +1,26 @@
 ﻿using System.Threading;
 using System.Threading.Tasks;
 using Raven.Client.Documents.Session;
+using Raven.Client.ServerWide.Operations;
 using Raven.Client.Util;
 using Sparrow.Json;
 
 namespace Raven.Client.Documents.Operations
 {
     public partial class OperationExecutor
     {
+        public Task<CmpXchgResult<T>> CompareExchangeAsync<T>(string name, T newValue, long index)
```

Comment:
Every operation uses Send or SendAsync methods. We do not need to create dedicated ones.

Reply:
Yes